tox.ini

```diff
@@ -17,8 +19,8 @@ deps =
     PyOpenGL
     pachi-py>=0.0.19
     box2d-py
-    doom_py>=0.0.11
-    mujoco_py<1.0.0,>=0.4.3
+    doom-py>=0.0.11
+    mujoco_py>=1.50.1,<1.50.2
```

Comment:
some gym code actually requires new version of mujoco and does not work with 0.5.x branch -> hence only python3 is tested. May need to test python3.6 separately.